.github/workflows/conda.yml

```
@@ -0,0 +1,78 @@
+name: Conda build
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+# When this workflow is queued, automatically cancel any previous running
+# or pending jobs from the same branch
+concurrency:
+  group: conda-${{ github.head_ref }}
+  cancel-in-progress: true
+
+# Required shell entrypoint to have properly activated conda environments
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  conda:
+    name: Build (and upload)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Python
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniforge-variant: Mambaforge
+          use-mamba: true
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          mamba install boa conda-verify
+
+          which python
+          pip list
+          mamba list
+      - name: Build conda packages
+        run: |
+          # suffix for pre-release package versions
+          export VERSION_SUFFIX=a`date +%y%m%d`
+
+          # latest dask-core pre-release build
+          export DASK_VERSION=`conda search --override-channels -c dask/label/dev dask-core | tail -n 1 | awk '{print $2}'`
+          export DASK_BUILD=`conda search --override-channels -c dask/label/dev dask-core | tail -n 1 | awk '{print $3}'`
```

Wonder if we can use [bash arrays]( https://www.gnu.org/software/bash/manual/html_node/Arrays.html ) to compute this once and extract each piece from that result

---

Perhaps something like this would work

```suggestion
          export DASK_INFO=( $(conda search --override-channels -c dask/label/dev dask-core | tail -n 1 | tr " " "\n") )
          export DASK_VERSION="${DASK_INFO[1]}"
          export DASK_BUILD="${DASK_INFO[2]}"`
```

---

Thanks @jakirkham 😄 was playing around with this myself, think we can get away with removing the `tr` command

---

Was thinking we should be able to avoid `tr`, but wasn't seeing how to do that on my Mac. Maybe Linux is better behaved 😄